march_hardware/test/joint_test.cpp

```diff
@@ -153,16 +153,16 @@ TEST_F(JointTest, GetTemperatureWithoutTemperatureGes)
 
 TEST_F(JointTest, ResetController)
 {
-  EXPECT_CALL(*this->imc, reset(_)).Times(1);
```

Comment:
I see you are using the IMC here as well as the IMC mock object, you can remove this and just mock the `MotorController` interface